src/SignalR/common/Shared/AsyncEnumerableAdapters.cs

```diff
@@ -24,13 +24,26 @@ public static IAsyncEnumerable<T> MakeCancelableTypedAsyncEnumerable<T>(IAsyncEn
             return new CancelableTypedAsyncEnumerable<T>(asyncEnumerable, cts);
         }
 
+#if NETCOREAPP3_0
         public static async IAsyncEnumerable<object> MakeAsyncEnumerableFromChannel<T>(ChannelReader<T> channel, [EnumeratorCancellation] CancellationToken cancellationToken = default)
         {
             await foreach (var item in channel.ReadAllAsync(cancellationToken))
```

Comment:
`channel.ReadAllAsync` isn't available on `netstandard2.1`, is this something we should ask CoreFX to provide or...?

Reply:
Yes

Reply:
We might get this for free with https://github.com/dotnet/corefx/issues/34963